README.md

```diff
@@ -3,3 +3,5 @@
 This repository contains sources that can be used in SUSE Manager image profiles to build various images.
 
 Repository is organized by build types and then individual image names.
+
+The special "Testsuites" directory contains profiles used by SUSE Manager cucumber testsuite.
```

Comment:
I'd create a list, explaining what each of the directories has:

```
Repository is organized by build types and then individual image names.

Build types, as directories at the root folder:

- `Containers`: Whatever
- `OSimages`: Whatever
- `Testsuites`: Profiles used by SUSE Manager cucumber testsuite.
```

Reply:
ok